### PR TITLE
upgrade active-fedora, solr_wrapper, and fcrepo_wrapper

### PR DESCRIPTION
**Pull request by carolyncole**
Generating active-fedora's new config files and modifying our yml files to point to those instances

### CODE REVIEW

File: config/solr.yml [hunk 1]
```diff
@@ -1,8 +1,7 @@
-# This is a sample config file that does not have multiple solr instances. You will also need to be sure to
-# edit the fedora.yml file to match the solr URL for active-fedora.
+# This is a sample config file that points to a solr server for each environment
 development:
-  url: http://127.0.0.1:8983/solr/hydra-development
+  url: http://127.0.0.1:<%= ENV['SOLR_TEST_PORT'] || 8983 %>/solr/hydra-development
```
Comment (awead) [inline config/solr.yml]: should this be SOLR_DEVELOPMENT_PORT?

File: tasks/dev.rake [hunk 2]
```diff
@@ -2,7 +2,7 @@ require 'fcrepo_wrapper'
 require 'fcrepo_wrapper/rake_task'
 require 'solr_wrapper'
 
-FcrepoWrapper.default_instance_options = { port: '8984', instance_dir: 'tmp/fcrepo' }
+#FcrepoWrapper.default_instance_options = { port: '8984', instance_dir: 'tmp/fcrepo' }
```
Comment (awead) [inline tasks/dev.rake]: We should be able to remove this now.

Comment (carolyncole) [inline tasks/dev.rake]: 🆗  will do

File: config/solr.yml (hunk 1 above)
Comment (carolyncole) [inline config/solr.yml]: 👍

**Comment by carolyncole** [comment]
@awead I think this is good to go now...

**Pull request merged**